tests/test_api_users.py

```diff
@@ -94,10 +94,6 @@ async def test_user_put(server):
     with pytest.raises(Exception):
         await client.request('PUT', '/api/users/test', {'loginShell': 'foo'})
 
-    await client.request('PUT', '/api/users/test', {'loginShell': ''})
-    ret = await krs.users.user_info('test', rest_client=krs_client)
-    assert ret['attributes']['loginShell'] == ''
-
```

Comment:
Sometime after version 15 Keycloak began to delete empty attributes (at least those not defined in user profile)